channels/es.m3u

```diff
@@ -101,16 +79,12 @@ https://5924d3ad0efcf.streamlock.net/canal4/canal4live/playlist.m3u8
 https://5940924978228.streamlock.net/Directo3/Directo3/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal10Emporda.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Canal 10 Empordà (360p) [Not 24/7]
 http://ventdelnord.tv:8080/escala/directe.m3u8
-#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Catalan" tvg-logo="" group-title="",Canal 24 horas (576p) [Geo-blocked]
-https://hlsliveamdgl8-lh.akamaihd.net/i/hlslive_1@156275/master.m3u8
-#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Catalan" tvg-logo="" group-title="",Canal 24 horas (576p) [Geo-blocked]
-https://hlsliveamdgl8-lh.akamaihd.net/i/hlslive_1@300207/master.m3u8
-#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Catalan" tvg-logo="" group-title="",Canal 24 horas [Geo-blocked]
-http://hlsliveamdgl8-lh.akamaihd.net/i/hlsdvrlive_1@583030/master.m3u8
-#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Catalan" tvg-logo="" group-title="",Canal 24 horas (1080p) [Timeout]
+#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/2084px-Logo_TVE-24h.svg.png" group-title="News",Canal 24 horas (1080p)
 https://rtvelivestreamv3.akamaized.net/24h_bkp_dvr.m3u8
-#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Catalan" tvg-logo="" group-title="",Canal 24 horas (720p) [Timeout]
+#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/2084px-Logo_TVE-24h.svg.png" group-title="News",Canal 24 horas (720p)
 https://rtvelivestreamv3.akamaized.net/24h_main_dvr.m3u8
+#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/2084px-Logo_TVE-24h.svg.png" group-title="News",Canal 24 horas (1080p)
+https://streamlink-iptv.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UC7QZIf0dta-XPXsp9Hv4dTw/live
```

Comment:
```suggestion
https://streamlink-iptv.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UC7QZIf0dta-XPXsp9Hv4dTw/live
#EXTINF:-1 tvg-id="Canal24horas.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/2084px-Logo_TVE-24h.svg.png" group-title="News",Canal 24 horas (1080p)
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UC7QZIf0dta-XPXsp9Hv4dTw/live
```
For the time being, add the other one as alternative. I know your Heroku follows my release, and @dct-infra managed to put the latest master of Streamlink, making it work.